src/swap/core/useGetETHBalance.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import { useBalance } from 'wagmi';
+import { formatUnits } from 'viem';
+import { getRoundedAmount } from '../../utils/getRoundedAmount';
+import type { Address } from 'viem';
+
+const ETH_DECIMALS = 18;
+export function useGetETHBalance(address: Address) {
```

Comment:
I think this hook should live in the `wallet` module. My thinking here is that we are checking the Balance of a Wallet.

And Swap uses this utility. Similar how Swap uses Token utilities.

Reply:
done !